### PR TITLE
chore: combine genesis tx to JSON file

### PR DESCRIPTION
**Pull request by jleach**
# Summary of Changes

To better automate keeping the genesis transaction up-to-date I'm propose merging them into a single JSON file that can easily be updated using the included JavaScript file `make-block.js`. This script pull down the genesis files from the ledger monitor repo and then imports all the latest genesis transactions.

# Related Issues

Please reference here any issue #'s that are relevant to this PR, or simply enter "N/A" if this PR does not relate to any existing issues.

# Pull Request Checklist

Tick all boxes below to demonstrate that you have completed the respective task. If the item does not apply to your this PR **check it anyway** to make it apparent that there's nothing to do.

- [x] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this);
- [x] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components;
- [x] Updated documentation as needed for changed code and new or modified features;
- [x] Added sufficient [tests](../__tests__/) so that overall code coverage is not reduced.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

Pro Tip 🤓

- Read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!
- Your PR will likely not be reviewed until all the above boxes are checked and all automated tests have passed.

_PR template adapted from the Python attrs project._

### CODE REVIEW

File: scripts/make-blocks.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+const fs = require('fs');
+const path = require('path')
+const https = require('https')
+const http = require('http')
+const url = require('url')
+
+const networkUrl = 'https://raw.githubusercontent.com/hyperledger/indy-node-monitor/main/fetch-validator-status/networks.json'
+const targetNetworks = [
+  { id: "bct", production: false },
```
Comment (amanji) [inline scripts/make-blocks.js:11]: Are these IDs guaranteed to be unique?

File: scripts/make-blocks.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+const main = async () => {
+  const networksAsString = await getUrlContents(url.parse(networkUrl))
+  const networks = JSON.parse(networksAsString)
+  const targets = targetNetworks.map((t) => Object({ ...t, ...networks[t.id] }))
+  let ledgers = []
+
+  for (let t of targets) {
+    const aUrl = url.parse(t['genesisUrl'])
```
Comment (amanji) [inline scripts/make-blocks.js]: Any insight into why it's named this way?

File: scripts/make-blocks.js (hunk 1 above)
Comment (amanji) [reply scripts/make-blocks.js:11]: My guess is this corresponds to the pulled network ids so probably, yes, they are likely to be unique.

Comment (jleach) [reply scripts/make-blocks.js:11]: I believe so.

File: scripts/make-blocks.js (hunk 2 above)
Comment (jleach) [reply scripts/make-blocks.js]: They're the networks I'm interested in being added to Bifold. I'll call it something more obvious.

**Review by amanji — APPROVED** [review]

**Pull request merged**